index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Asap:wght@500&family=Poppins:wght@100;800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- Nav Bar  -->
+
+    <header class="nav-bar">
+      <div class="header-logo">
+        <a href="#" class="header-logo-link"><h3>My Logo</h3></a>
+      </div>
+
+      <button type="button" class="nav-button">
+        <img
+          src="./styling-features-pictures/Union.png"
+          alt="nav-button-menu"
+        />
+      </button>
+    </header>
+
+    <!-- Headline -->
+
+    <!-- <div class="special-background"> -->
+    <section class="headline">
+      <div class="intro">
+        <h1>I’m Sebastian. Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+
+        <div class="connect-div">
+          <a href="#" class="connect-a"><h3>Let's Connect</h3></a>
+        </div>
+
+        <div class="social-icons">
+          <ul>
+            <li><i class="fa fa-instagram"></i></li>
+            <li><i class="fa fa-twitter"></i></li>
+            <li><i class="fa fa-github"></i></li>
+            <li><i class="fa fa-linkedin"></i></li>
+          </ul>
```

Comment:
- [ ] Kindly wrap your icon with `a` tag, to be link to your accounts.

like
```
<ul>
  <li><a href=""><i....><i/><a/></li>
  ............
</ul>
```

Reply:
Hi, I Appreciate your suggestions.
Would you be so kind of helping me to know why the hover is not working on these links nor in the "let's get it" button? because it's working on the navbar buttons.